.github/workflows/GHPages.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy to GitHub Pages
+
+on:
+    push:
+        branches: [main]
+    workflow_dispatch: null
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+    contents: read
+    pages: write
+    id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+    group: pages
+    cancel-in-progress: true
+
+jobs:
+    deploy:
+        environment:
+            name: github-pages
+            url: ${{ steps.deployment.outputs.page_url }}
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: 18
+            - name: Install Packages
+              run: npm ci
+            - name: Build docs
+              run: |+
+                  export NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
Currently we need this workaround to use vuepress (v1). If we want to remove the workaround, the simplest replacement would be VuePress (v2) or VitePress, neither of which is GA yet.

https://v2.vuepress.vuejs.org/
https://vitepress.vuejs.org/